.docker/Makefile

```diff
@@ -0,0 +1,77 @@
+# The following is a temporary build script until a Concourse CI pipeline exists
+# It is just designed for building a production ready image using --target=web
+# To use these commands you will need AWS access and your profiles configured correctly
```

Comment:
q: (to confirm) is the idea this will be run manually?

also, suggestion: AWS cli is a dependency too, so perhaps add a note about it?

Reply:
This is very temporary, so I don't want to sink too much time into perfecting this.

For now, yes it will be run manually. The AWS CLI is only needed for deployment, which should happen outside the container anyway.